models/articles_models.js

```diff
@@ -10,6 +10,14 @@ exports.fetchArticleByID = (id) => {
         }
         return rows[0];
     })
+    .then((article) => {
+        return db.query("SELECT COUNT(*) FROM comments WHERE article_id = $1", [id])
+        .then(({ rows }) => {
+            const numComments = Number(rows[0].count)
+            article.comment_count = numComments;
+            return article;
+        })
```

Comment:
Would be better if it only queries the database once instead of twice with different table. Using Join to join two tables can solve this problem